pkg/names/names.go

```diff
@@ -82,13 +82,19 @@ var (
 		{"Bgp", "BGP", "bgp", nil},
 		{"Cors", "CORS", "cors", nil},
 		{"Cidr", "CIDR", "cidr", nil},
+		{"Cname", "CNAME", "cname", nil},
 		{"Cpu", "CPU", "cpu", nil},
+		{"Crl", "CRL", "crl", nil},
+		{"Cps", "CPS", "cps", nil},
+		{"Csr", "CSR", "csr", nil},
 		{"Dhcp", "DHCP", "dhcp", nil},
 		{"Dns", "DNS", "dns", nil},
 		{"Dpd", "DPD", "dpd", nil},
 		{"Ebs", "EBS", "ebs", nil},
 		{"Ec2", "EC2", "ec2", nil},
 		{"Ecr", "ECR", "ecr", nil},
+		// Prevent "Edit" from becoming "EDIt"
+		{"Edi", "EDI", "edi", regexp.MustCompile("Edi(?!t)", regexp.None)},
```

Comment:
Mind adding a quick unit test that verifies this one?

Reply:
Added! And sorted all tests because their ordering was annoying me